<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>621</width>
     <height>16777215</height>
    </size>
   </property>
   <column>
    <property name="text">
     <string>Проекты</string>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Мои проекты</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton33">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton333">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton30">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
